fe/fe-core/src/main/java/com/starrocks/qe/ShowExecutor.java

```diff
@@ -408,7 +408,9 @@ private void handleShowMaterializedView() throws AnalysisException {
                 if (GlobalStateMgr.getCurrentState().isUsingNewPrivilege()) {
                     AtomicBoolean baseTableHasPrivilege = new AtomicBoolean(true);
                     mvTable.getBaseTableInfos().forEach(baseTableInfo -> {
-                        if (!PrivilegeManager.checkTableAction(connectContext, db.getFullName(),
+                        Table baseTable = baseTableInfo.getTable();
+                        if (baseTable != null && baseTable.isLocalTable() && !PrivilegeManager.
```

Comment:
Add TODO comments to prevent forgetting this when supporting external permissions in the future

Reply:
Done